metrics/coverage_converter/test/common/exception/coverage_converter_exception_test.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:test/test.dart';
+
+import '../../../lib/common/exception/coverage_converter_error_strings.dart';
+import '../../../lib/common/exception/coverage_converter_exception.dart';
+import '../../../lib/common/exception/error_code/coverage_converter_error_code.dart';
+
+void main() {
+  group("CoverageConverterException", () {
+    test(
+      "maps the 'no such file' error code to the 'no such file' error message",
```

Comment:
Please, start the test description with the `.methodName` (see [dart code style](https://github.com/platform-platform/monorepo/blob/master/docs/10_dart_code_style.md#test-descriptions)).
```suggestion
      ".message returns no such file error message if the given code is `noSuchFile`",
```